doc/readme.md

```diff
@@ -0,0 +1,16 @@
+# Trapperkeeper Authorization
```

Comment:
I don't have any problem with the existence of this file.  If @nathanielksmith has found it helpful in other repos, I don't see any reason to remove it.  However, I share some of @camlow325's concerns about either duplicating content here or getting into a split-brain situation where it's not clear if things should live in this file or the top-level README.md.  So I +1 @jpinsonault's comment about linking to the top-level README as well as perhaps deleting some of the text in this file which explains what this project is/isn't, etc. (as that should be covered in the top-level README) and leave this doc as just a sort of "meta-doc".